### PR TITLE
[DE-248] DE-433 Cancel/Update/Override subscription

### PR DESCRIPTION
**Pull request by michalpierog**

### CODE REVIEW

File: components/schemas/Update-Subscription-Component.yaml [hunk 1]
```diff
@@ -0,0 +1,7 @@
+title: Update Subscription Component
+type: object
+properties:
+  component_id:
+    type: integer
+  custom_price:
+    $ref: "./Component-Custom-Price.yaml"
```
Comment (maciej-nedza) [inline components/schemas/Update-Subscription-Component.yaml:7]: Maybe leave description `Create or update custom pricing unique to the subscription`
I don't fully understand description between this and `Used in place of price_point_id to define a custom price point unique to the subscription` but it sounds a little different

**Review by maciej-nedza — APPROVED** [review]
LGTM

File: reference/Chargify-API.v1.yaml [hunk 2]
```diff
@@ -21257,14 +21244,6 @@ servers:
 security:
   - BasicAuth: []
 components:
-  parameters:
-    subscription-id-required-path:
-      name: subscription_id
-      in: path
-      description: The Chargify id of the subscription
-      schema:
-        type: integer
```
[the remark this answers is not in the document]
Comment (michalpierog) [reply reference/Chargify-API.v1.yaml:21266]: This is path variable so it doesn't matter much, but I think it should be integer everywhere. We can use String as in other places as it's more generic, but in the future we might change all ids to integers.

[the remark this answers is not in the document]
Comment (maciej-nedza) [reply reference/Chargify-API.v1.yaml:21266]: Ok, I meant that `subscription-id-path.yaml`, which now is used instead of that one is of type string

**Pull request merged**